File: Data Structures & Algorithms/max-water-container/submission-0.cpp
class Solution {
public:
    int maxArea(vector<int>& heights) {
        int ans = 0,t;
        for(int i=0;i<heights.size()-1;i++) {
            for(int l=i+1;l<heights.size();l++){
                t = min(heights[i],heights[l])*(l-i);
                ans = t>ans ? t : ans;
            }
        }
        return ans;
    }
};

File: Data Structures & Algorithms/buy-and-sell-crypto/submission-0.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        MIN = None
        PROFIT = 0
        for x in prices:
            if MIN == None or x<MIN:
                MIN = x
            else:
                temp = x - MIN
                PROFIT = temp if temp > PROFIT else PROFIT
        return PROFIT
